docs/source/get_started/credentials.rst

```diff
@@ -7,7 +7,10 @@ Below are instructions for accessing Superstaq credentials.
 
 #. Navigate to https://superstaq.infleqtion.com/.
 #. Choose your preferred method to sign up or sign in (Google or other email).
-#. Click the clipboard symbol to copy the key shown next to "Your API key". Save this key in a secure location; never upload your API key to GitHub.
+#. Verify your account by following the link that you receive in the confirmation email.
```

Comment:
I think it's probably useful to add something to the effect of "You will not be able to compile/submit jobs without verifying your email".